### PR TITLE
fix: Allow disabling PipelineActivity update check

### PR DESCRIPTION
**Pull request by abayer**
Specifically for use with static master BDD tests. Also cleans up some
debug logging I forgot to delete.

Signed-off-by: Andrew Bayer <andrew.bayer@gmail.com>

### CODE REVIEW

File: test/helpers/test.go [hunk 1]
```diff
@@ -461,11 +463,12 @@ func (t *TestOptions) CreatePullRequestAndGetPreviewEnvironment(statusCode int)
 		buildNumber = t.ThereShouldBeAJobThatCompletesSuccessfully(jobName, TimeoutBuildCompletes)
 		utils.ExpectNoError(err)
 	})
-	By("verifying that PipelineActivity has been updated to include the pull request title", func() {
-		pullTitle := t.GetPullTitleFromActivity(owner, applicationName, "pr-" + strconv.Itoa(prNumber), buildNumber)
-		Expect(pullTitle).Should(Equal(prTitle))
-	})
-
+	if t.ShouldTestPipelineActivityUpdate() {
```
Comment (hferentschik) [inline test/helpers/test.go]: So on static master the pipeline activity is not updated then, interesting.

**Comment by abayer** [comment]
/hold

Holding so I can verify that the test still actually gets run here in the logs.

**Pull request merged**